cmd/drone-docker/main.go

```diff
@@ -103,6 +103,11 @@ func main() {
 			Usage:  "docker daemon executes in debug mode",
 			EnvVar: "PLUGIN_DEBUG,DOCKER_LAUNCH_DEBUG",
 		},
+		cli.BoolFlag{
```

Comment:
change to `cli.BoolTFlag` so that it defaults to true

Reply:
done